asv.conf.json

```diff
@@ -1,16 +1,9 @@
 {
-    "version": 1,
-    "project": "Mitiq",
-    "project_url": "https://github.com/unitaryfund/mitiq",
-    "repo": ".",
-    "branches": ["master"],
-    "dvcs": "git",
-    "environment_type": "virtualenv",
-    "show_commit_url": "https://github.com/unitaryfund/mitiq/commit/",
-    "pythons": ["3.7"],
-     "benchmark_dir": "benchmarks",
-    "env_dir": ".asv/env",
-    "results_dir": ".asv/results",
-    "html_dir": ".asv/html",
-    "hash_length": 7
```

Comment:
Any reason to axe these?

Reply:
Not **great** reasons, but here are some.
- `benchmarks` is the default for `benchmark_dir`
- `hash_length` defaults to 8
- Since `asv` will primarily run via github actions, the custom `env`/`results`/`html` directories don't seem to provide any benefit
- On second thought, `pythons` might be worth keeping just to keep explicit track of what python version we are using to run the benchmarks

If this is a [chesterton's fence](https://fs.blog/chestertons-fence/) scenario, sorry!

Reply:
There wasn't really strong asv rationale behind including these - just for sake of being explicit. I would definitely encourage being explicit about the python version, but the rest sounds good.